### DISCUSSION TITLE
Custom accordian

### DISCUSSION THREAD

**Discussion by tresero** [razonyang/hugo-theme-bootstrap#771] [Q&A] [answered]
I wonder if there is a way to easily add a mailing list signup form in an accordian, or just modify the author accordian?

This could be sitewide, or just per content type.

**Comment by razonyang** [comment, marked as answer]
Hi, I guess you mean adding custom widget in the sidebar, please correct me if wrong. If so, it can be achieved by hooks.

1. Create a widget partial named `layouts/partials/custom/signup-form.html` relative to your site root.

```html
<div class="row component card">
    <div class="card-body">
        MY MAILING SIGNUP FORM
    </div>
</div>
```

2. And then import the partial via `layouts/partials/hooks/sidebar-begin.html`(create if the file does not exist).

```html
{{- partial "custom/signup-form" . }}
```

![image](https://user-images.githubusercontent.com/17720932/200981490-ba9bbcaa-c95a-43e7-8d2a-7ca129a1614c.png)

See also [Hooks](https://hbs.razonyang.com/v1/en/docs/advanced/hooks/) and [Accordion](https://getbootstrap.com/docs/5.2/components/accordion/).

**Reply by tresero** [reply]
Thanks, I'm new to Hugo and Go. I have a lot to learn!
